packages/perseus/src/perseus-types.js

```diff
@@ -1404,7 +1415,7 @@ export type PerseusInputNumberWidgetOptions = {|
     rightAlign?: boolean,
     simplify: "required" | "optional" | "enforced",
     size: "normal" | "small",
-    value: string | number | boolean,
+    value: string | number,
```

Comment:
Part of the code was doing `parseFloat` on value. This returns a number for `10` and `"10"`, but returns NaN for boolean. I could be wrong, but it seemed unlikely that we would want that.